cttportfolio.css

```diff
@@ -1,470 +1,874 @@
+
+
 * {
 	box-sizing: border-box;
 }
-html{
-	font-size:62.5%;
-	font-family:'Montserrat', sans-serif;
-	color:rgba(4,8,9,1);
+
+html {
+	font-size: 62.5%;
+	font-family: 'Montserrat', sans-serif;
+	color: rgba(4, 8, 9, 1);
+	border-radius: 7px;
 }
-body{
-	margin:0;
-	background-color:rgb(2,8,9);
+
+body {
+	margin: 0;
+	background-color: rgb(2, 8, 9);
 }
-main > div{
-	margin:0 auto;
-	max-width:1300px;
+
+main>div {
+	margin: 0 auto;
+	max-width: 1300px;
+}
+
+main {
+	margin-top: 200px;
 }
-h4, h5, h6,.pp_body h1{
-	font-weight:700;
-	font-size:6.0rem;
-	text-transform:uppercase;
+
+h4,
+h5,
+h6,
+.pp_body h1 {
+	font-weight: 700;
+	font-size: 6.0rem;
+	text-transform: uppercase;
 	-webkit-text-stroke-width: 0.3px;
 	-webkit-text-stroke-color: black;
-	color: rgb(248,248,255);
+	color: rgb(248, 248, 255);
+}
+
+h5 {
+	font-weight: 600;
+	font-size: 4rem;
 }
-h5{
-	font-weight:600;
-	font-size:4rem;
+
+.am_text,
+button,
+.navlinks a,
+.contactmetext>div,
+img {
+	box-shadow: 0px -2px 5px rgba(2, 8, 9, 0.6)
 }
-.am_text, button, .navlinks a, .contactmetext > div, img{
-	box-shadow: 0px -2px 5px rgba(2,8,9,0.6)
+
+button {
+	color: white;
+	background: none;
+	padding: 10px;
+	font-size: 2rem;
+	text-transform: uppercase;
+	border: 1px solid rgb(248, 248, 255);
+	font-weight: 600;
+	border-radius: 5px;
 }
-button{
-	color:white;
-	background:none;
-	padding:10px;
-	font-size:2rem;
-	text-transform:uppercase;
-	border:1px solid rgb(248,248,255);
-	font-weight:600;
-	border-radius:10px;
+
+a.icon_hover img,
+a.pf_sub_links h6,
+.navlinks.icon_hover>a:hover {
+	border: 1px solid rgba(248, 248, 255, 0);
```

Comment:
This doesn't need to be `rgba()`, an alpha of `0` is the same as just using `transparent`.